build/db_schemas/yugabyte/rid/upto-v1.0.0-crdb_v4.0.0.sql

```diff
@@ -0,0 +1,45 @@
+-- This migration is equivalent to rid v4.0.0 schema for CockroachDB.
+
+CREATE TABLE subscriptions (
+    id UUID PRIMARY KEY,
+    owner TEXT NOT NULL,
+    url TEXT NOT NULL,
+    notification_index INT4 DEFAULT 0,
+    starts_at TIMESTAMPTZ,
+    ends_at TIMESTAMPTZ,
+    updated_at TIMESTAMPTZ NOT NULL,
+    cells BIGINT[] NOT NULL,
+    writer TEXT,
+    CHECK (starts_at IS NULL OR ends_at IS NULL OR starts_at < ends_at),
+    CHECK (array_length(cells, 1) IS NOT NULL)
+);
+CREATE INDEX s_owner_idx ON subscriptions (owner);
+CREATE INDEX s_starts_at_idx ON subscriptions (starts_at);
+CREATE INDEX s_ends_at_idx ON subscriptions (ends_at);
+CREATE INDEX s_cell_idx ON subscriptions USING ybgin (cells);
+CREATE INDEX subs_by_time_with_owner ON subscriptions (ends_at) INCLUDE (owner);
+
+CREATE TABLE identification_service_areas (
+    id UUID PRIMARY KEY,
+    owner TEXT NOT NULL,
+    url TEXT NOT NULL,
+    starts_at TIMESTAMPTZ,
+    ends_at TIMESTAMPTZ,
+    updated_at TIMESTAMPTZ NOT NULL,
+    cells BIGINT[],
```

Comment:
This is missing `NOT NULL` constraint,  while the `cells` on `subscriptions` has it. 
I'm not sure I understand that part of the crdb migrations:
https://github.com/interuss/dss/blob/ed31e91c617fe23b9edeab3a2ca2e885ba7d34f4/build/db_schemas/rid/upto-v2.0.0-add_inverted_indices.sql#L38-L43

If at the end we are supposed to have or not a `NOT NULL` constraint, but in any case it looks like it should be consistent between `subscriptions` and `operations`.